src/main/resources/db/migration/metadata/V2__update_idl_types.sql

```diff
@@ -1,10 +1,6 @@
-ALTER TYPE idl ADD VALUE 'openapiyaml';
-
-UPDATE metaprotocols SET idl_name = 'openapiyaml' WHERE idl_name = 'openapi';
-
 ALTER TYPE idl RENAME TO idl_old;
 
-CREATE TYPE idl as ENUM('avro', 'protobuf', 'mu', 'openapiyaml', 'openapijson', 'scala');
+CREATE TYPE idl as ENUM('avro', 'protobuf', 'mu', 'openapi', 'openapijson', 'scala');
```

Comment:
Should we then remove 'openapijson' here?
```suggestion
CREATE TYPE idl as ENUM('avro', 'protobuf', 'mu', 'openapi',  'scala');
```

Reply:
This is against your branch. The idea is to just add the new `openapijson` type. If we keep `openapi` (instead of `openapiyaml`) we don't need to apply a `ALTER TYPE( .*)? ADD VALUE` statement. Does it make sense?

Reply:
ooook yeah, I got you now!